hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupBasedLoadBalancer.java

```diff
@@ -77,6 +81,23 @@
   private volatile RSGroupInfoManager rsGroupInfoManager;
   private LoadBalancer internalBalancer;
 
+  /** Define the config key of if fallback group is enabled */
+  public static final String FALLBACK_GROUP_ENABLED_KEY = "hbase.rsgroup.fallback.groups.enable";
+  public static final boolean FALLBACK_GROUP_ENABLED_DEFAULT = false;
+
+  /** Define the config key of fallback groups */
+  public static final String FALLBACK_GROUPS_KEY = "hbase.rsgroup.fallback.groups";
+  public static final String FALLBACK_GROUPS_DEFAULT = RSGroupInfo.DEFAULT_GROUP;
+
+  /** Define the config key of interval to correct fallback regions */
+  static final String FALLBACK_CORRECT_INTERVAL_KEY = "hbase.rsgroup.fallback.correct.interval";
+  static final int FALLBACK_CORRECT_INTERVAL_DEFAULT = 10 * 60 * 1000; // 10min
+
+  private boolean fallbackEnabled;
+  private List<String> fallbackGroups;
+  @VisibleForTesting
+  FallbackCorrector fallbackCorrector;
```

Comment:
Do we need this chore? This should be work of RSGroupBalancer?

Reply:
I tried and make sure that RSGroupBalancer works for this when the balancer switch is on. Let me remove the code about the chore.